File: quark/test/emit/autocast/py/autocast.py
import os, sys

_Map = dict
_List = list

def _println(obj):
    if obj is None:
        sys.stdout.write("null\n")
    else:
        sys.stdout.write("%s\n" % obj)


def test():
    o = "Hello World!";
    s = o;
    s2;
    s2 = o

File: quark/test/emit/autocast/js/autocast.js
var _qrt = require("quark_runtime.js");

function foo(s) {
    _qrt.print(s);
}
exports.foo = foo;

function main() {
    var o = "Hello World!";
    var s = o;
    var s2;
    s2 = o;
    foo(o);
}
exports.main = main;

main();
